app/utils/simplified-account/validation/responsible-person.schema.js

```diff
@@ -0,0 +1,146 @@
+const { body } = require('express-validator')
+const { formatPhoneNumberWithCountryCode } = require('../../telephone-number-utils')
+
+const responsiblePersonSchema = {
+  name: {
+    firstName: {
+      validate: body('firstName')
+        .notEmpty()
+        .withMessage('Enter your first name')
+        .bail()
+        .isLength({ max: 100 })
+        .withMessage('First name must be 100 characters or fewer')
+    },
+    lastName: {
+      validate: body('lastName')
+        .notEmpty()
+        .withMessage('Enter your last name')
+        .bail()
+        .isLength({ max: 100 })
+        .withMessage('Last name must be 100 characters or fewer')
+    }
+  },
+  dob: {
+    validate: body('dobDay') // Use just one DOB field as the anchor to prevent multiple validations
+      .custom((value, { req }) => {
+        const { dobDay, dobMonth, dobYear } = req.body
+        if (!dobDay && !dobMonth && !dobYear) {
+          throw new Error('Enter the date of birth')
+        }
+
+        // Stripe requires minimum age of 13
+        const day = Number(dobDay)
+        const month = Number(dobMonth) - 1
+        const year = Number(dobYear)
+
+        const dob = new Date(year, month, day)
```

Comment:
Months seems to be zero-based in Node 😬 Furthermore dates roll over if an invalid day of month is given 😬 
```> new Date(1999, 02, 31)
1999-03-30T23:00:00.000Z
> new Date(1999, 01, 31)
1999-03-03T00:00:00.000Z
> new Date(1999, 01, 01)
1999-02-01T00:00:00.000Z
> new Date(1999, 0, 01)
1999-01-01T00:00:00.000Z
> new Date(1999, 01, 32)
1999-03-04T00:00:00.000Z
```
I think we need a test for `responsible-person.schema.js`. But happy for this to go in a different PR.

Reply:
yeah I minus the month on line 33

Reply:
Do you plan to do further testing around the validations?

Reply:
yes, this PR is quite large. I have cypress and controller tests coming shortly